apps/server/src/db/schema/comments.ts

```diff
@@ -6,24 +6,35 @@ import {
   text,
   timestamp,
   uniqueIndex,
+  varchar,
 } from "drizzle-orm/pg-core";
 
 import { tastings } from "./tastings";
 import { users } from "./users";
 
+// Maximum length for comment text
+const MAX_COMMENT_LENGTH = 2000;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const comments = pgTable(
-  // oops named this wrong sorry
   "comments",
   {
     id: bigserial("id", { mode: "number" }).primaryKey(),
     tastingId: bigint("tasting_id", { mode: "number" })
       .references(() => tastings.id)
       .notNull(),
-    comment: text("comment").notNull(),
+    comment: varchar("comment", { length: MAX_COMMENT_LENGTH }).notNull(),
     createdAt: timestamp("created_at").defaultNow().notNull(),
     createdById: bigint("created_by_id", { mode: "number" })
       .references(() => users.id)
       .notNull(),
+    // Add parentId for replies with ON DELETE CASCADE
+    parentId: bigint("parent_id", { mode: "number" }).references(
+      () => comments.id,
+      { onDelete: "cascade" },
+    ),
+    // Add mentions field to store mentioned usernames with a length constraint
+    mentions: varchar("mentions", { length: 500 }),
```

Comment:
look at tags schema for an example of how to use a varchar array (not sure if my default is right there)